.circleci/config.yml

```diff
@@ -308,6 +308,11 @@ workflows:
           module: fxa-oauth-console
           requires:
             - install
+      - build-module:
+          name: fxa-payments-server
+          module: fxa-payments-server
+          requires:
+            - install
```

Comment:
Just noting this step doesn't do anything useful right now because `.circleci/test.sh` looks for a `Dockerfile-test` that doesn't exist in this case. Should we delete this step and just keep the `deploy-module` one instead, to save resources in CI? Or is a `Dockerfile-test` implementation coming later?